lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -992,6 +995,10 @@ def view_init(self, elev=None, azim=None, vertical_axis="z"):
             The azimuth angle in the horizontal plane in degrees.
             If None then the initial value as specified in the `Axes3D`
             constructor is used.
+        roll : float, default: None
+            The roll angle about the viewing direction in degrees.
```

Comment:
I'd suggest that we define exactly what these mean and the order.  The previous two angles should be defined as well

```
azimuth : float, default: None
    view angle starting from looking along the negative y axis in degrees, positive moves the viewing
    angle around the z axis in the counter-clockwise direction. ...

elevation : float, default: None
    view elevation, starting with the view lined up with the x-y plane, positive in degrees moves the view 
    above the x-y plane. ....
```
```suggestion
            After azimuth and elevation have been applied, the roll angle about the 
            viewing direction in degrees.  Positive spins the scene counter-clockwise.
```

Reply:
Done. I did not include the rotation order because that is an implementation detail the user doesn't need to think about, and I think it would just add confusion. The camera movement is unambiguous as defined.